text/delegatable_cdd/README.md

```diff
@@ -0,0 +1,514 @@
+- RFC PR: (leave this empty)
+- Ursa PR: https://github.com/hyperledger/ursa/pull/51
+- Version: (use whole numbers: 1, 2, 3, etc)
+
+# Summary
+[summary]: #summary
+
+This is an implementation of delegatable anonymous credentials as described in
+the CCS 17 paper [Practical UC-Secure Delegatable Credentials with Attributes
+and Their Application to
+Blockchain](https://acmccs.github.io/papers/p683-camenischA.pdf) by Camenisch
+et.al. This allows issuers to delegate issuance rights to other entities such
+that the presentations from their issued credentials do not reveal their
+identities but only the identity of the the first issuer in the delegation chain.
+This also allows issuers at each level to add attributes in their issued
+credentials. However, downstream issuers in the delegation do learn about all
+the upstream issuers, i.e. there is no privacy among issuers. This
+implementation is not UC-secure but only implements the delegatable credentials
+from the paper.
+
+# Motivation
+[motivation]: #motivation
+
+Delegatable credentials are useful in saving operational costs of issuance and
+can also make issuance cheaper. If an organization that wants to issue
+credentials has only the head of the organization (or the relevant individual)
+issuing the credential, it becomes a bottleneck. But if the head decides to
+delegate the issuance rights to its subordinates, then the credential receivers
+can check that credential comes from a delegatee of the head. In addition,
+delegation provides privacy to issuers who might want to remain anonymous to
+verifiers. This [Aries RFC](https://github.com/hyperledger/aries-rfcs/pull/218)
+goes into more details of delegatable credentials.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+The entity delegating the credential issuance rights and identified to the
+verifier is called the *root issuer* or the *root delegator*. All issuers have a
+signing and verification key but only the root issuer's verification key is
+known to the verifier. The linear structure starting at the root issuer and
+consisting of subsequent issuers involved in delegation is called the *issuer
+chain*. For reasons explained later, an issuer is either an odd-level issuer or
+an even-level issuer depending on its position in the issuer chain. The *root
+issuer* is always an even-level issuer. Similarly, the linear structure starting
+at the root issuer's credential and consisting of subsequent issuers'
+credentials is called the *credential chain*. A credential is either an odd
+level credential or even-level credential depending on the issuer; an even-level
+issuer always issues an odd-level credential and vice versa. Each credentials in
+the credential chain is a also called a *link* and has a level which is a
+monotonically increasing number starting from 1 with no gaps. The credential
+(or link) issued by the root issuer has level 1, the subsequent link will have level
```

Comment:
"issued by the root issuer has level 1" --> "issued by the root issuer has level 1, conferring on the downstream recipient an odd-level status" (I'm trying to clarify that the level of a credential is associated with the level of its holder, not the level of its issuer)